sytest/pipeline.yml

```diff
@@ -185,37 +185,145 @@ steps:
         - exit_status: 2
           limit: 2
 
-  - label: "Dendrite - :go: 1.11 / :postgres: 9.6"
-    agents:
+  - label: "Dendrite - :go: 1.13 / :postgres: 9.6"
+      agents:
```

Comment:
this looks extremely bogus. why are you changing the indentation here?

Reply:
sigh, because VS Code is awful. That is unintentional. :/

Reply:
use a proper editor :innocent:

Reply:
This is hopefully less bad now!